test/unit/features.spec.js

```diff
@@ -0,0 +1,26 @@
+describe('features', function() {
+
+  beforeEach(function() {
+    _.extend(Marionette.FEATURES, {
```

Comment:
as long as we're careful not to introduce a 'foo' feature tag later :stuck_out_tongue::